themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
@pmario, I agree that this PR is not the place to kind sneak this change into the core ;-), I'd also be in favor of using the the 0.5em padding of hljs, as I think code-samples should stand out a little. After all, that's why they're there.